pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1281,13 +1281,46 @@ func (self *AWSCloud) TCPLoadBalancerExists(name, region string) (bool, error) {
 	return false, nil
 }
 
+// Retrieves instance's vpc id from metadata
+func (self *AWSCloud) findVPCId() (string, error) {
```

Comment:
I am not sure what to do about this method. My reason why I want to keep it in a separate method is to be able to test it easier...

At least I renamed the method to respect the style guide